spring-cloud-stream-core-docs/src/main/asciidoc/preface.adoc

```diff
@@ -1,3 +1,19 @@
+== A Brief History of Spring's Data Integration Journey
+
+Spring's journey on Data Integration started with https://projects.spring.io/spring-integration/[Spring Integration]. With its programming model, it provided a consistent developer experience to build applications that can embrace http://www.enterpriseintegrationpatterns.com/[Enterprise Integration Patterns] to connect with external systems such as databases, message brokers, and among others.
+
+Fast forward to cloud-era, where microservices were prominent in the enterprise setting. https://projects.spring.io/spring-boot/[Spring Boot] transformed the way how developers built Applications. With Spring's programming model and the runtime responsibilities handled by Spring Boot, it became seamless to develop standalone production grade Spring-based microservices.
+
+To extend this to Data Integration workloads, Spring Integration and Spring Boot were put together into a new project. Spring Cloud Stream was born.
+
+With Spring Cloud Stream, developers can:
+* Build, test, iterate, and deploy data-centric applications in isolation
+* Apply modern microservices architecture patterns, including composition via messaging
+* Decouple application responsibilities with event-centric thinking. An event can represent something that has happened in time, to which the downstream consumer applications can react without knowing where it originated or the producer identity
+* Port the business logic onto message brokers of choice (e.g., RabbitMQ, Apache Kafka, Amazon Kinesis)
```

Comment:
Use this: "Port the business logic onto message brokers (such as RabbitMQ, Apache Kafka, Amazon Kinesis)"